vars/withDefinitionImportSecrets.groovy

```diff
@@ -0,0 +1,26 @@
+
+def call(String product, String environment, Map<String, String> vaultOverrides, Closure block) {
+  def secrets = [
+    [$class: 'AzureKeyVaultSecret', secretType: 'Secret', name: 'definition-importer-username', version: '', envVariable: 'DEFINITION_IMPORTER_USERNAME'],
+    [$class: 'AzureKeyVaultSecret', secretType: 'Secret', name: 'definition-importer-password', version: '', envVariable: 'DEFINITION_IMPORTER_PASSWORD'],
+    [$class: 'AzureKeyVaultSecret', secretType: 'Secret', name: 'ccd-api-gateway-oauth2-client-secret', version: '', envVariable: 'CCD_API_GATEWAY_OAUTH2_CLIENT_SECRET'],
+    [$class: 'AzureKeyVaultSecret', secretType: 'Secret', name: 'ccd-api-gateway-s2s-key', version: '', envVariable: 'CCD_API_GATEWAY_S2S_KEY'],
+  ]
+
+  String theKeyVaultUrl = getKeyVaultUrl(product, environment, vaultOverrides)
+
+  withAzureKeyvault(
+    azureKeyVaultSecrets: secrets,
+    keyVaultURLOverride: theKeyVaultUrl,
+    applicationIDOverride: env.AZURE_CLIENT_ID,
+    applicationSecretOverride: env.AZURE_CLIENT_SECRET
```

Comment:
not needed, we don't use the override and it's just confusing as they're empty strings / null
```suggestion
```

Reply:
I have used these 2 params, because I followed [withTeamSecrets](https://github.com/hmcts/cnp-jenkins-library/blob/c00688e9f2cca1ac5b6161fe8dda05af33c3ff7d/vars/withTeamSecrets.groovy#L23) and they are passed to the method. So, if we don't need them I will remove.

Reply:
don't need them, that can be fixed separately.

Reply:
fixed in [commit 24826c18d969b3ac35100fa7b2b9607edb49aa86](https://github.com/hmcts/cnp-jenkins-library/pull/747/commits/24826c18d969b3ac35100fa7b2b9607edb49aa86)